libethashseal/GenesisInfo.h

```diff
@@ -34,10 +34,10 @@ namespace eth
 enum class Network
 {
 	//Olympic = 0,			///< Normal Olympic chain.
-	Frontier = 1,			///< Normal Frontier/Homestead chain.
-	//Morden = 2,				///< Normal Morden chain.
+	MainNetwork = 1,		///< Normal Frontier/Homestead/DAO/EIP150/EIP158 chain.
+	//Morden = 2,			///< Normal Morden chain.
 	Ropsten = 3,			///< New Ropsten Test Network
-	Test = 70,				///< Normal Frontier/Homestead chain without all the premine.
+	TransitionnetTest = 70,	///< Normal Frontier/Homestead/DAO/EIP150/EIP158 chain without all the premine.
```

Comment:
to do it this way we might need to add a compilation flag to libethashseal so it would include files from cpp-ethereum/test